### ISSUE TITLE
wd not found in second Terminal tab using caching

### ISSUE THREAD

**Issue by shoeffner** [zsh-users/antigen#391]
Error: `wd:.:1: no such file or directory: /plugins/wd/wd.sh`

I can reproduce the following issue:
1. Open up a terminal.
1. Run `antigen reset` or `antigen update`.
1. Open a new tab or terminal.
1. Run `wd d`: Success.
1. Open a new tab or terminal.
1. Run `wd d`: `wd:.:1: no such file or directory: /plugins/wd/wd.sh`
1. Go back to 2.

My simplified test files:

`.zshrc`:
```
# ANTIGEN
source /usr/local/opt/antigen/share/antigen/antigen.zsh
antigen init ${HOME}/.antigenrc
```

`.antigenrc`:
```
antigen use oh-my-zsh
antigen bundle wd
antigen apply
```

When I move the contents of `.antigenrc` to `.zshrc` it seems to work, however despite the praised performance boost it also feels cleaner to have a different file to declare all bundles, so I would love to stick to that approach.

Metadata:
macOS Sierra 10.12.2
Terminal.app
zsh 5.3.1 (x86_64-apple-darwin16.3.0)
Antigen v1.3.3

I set up everything using homebrew.

**Comment by desyncr** [comment]
It seems an issue with how current caching implementation works.
We are testing a different approach to it that will solve this issue and a few others there are.

An option you currently have is disable caching for the time being.

Sorry for the inconvenience and thanks for reporting!

DC*
Sent from mobile.

**Comment by desyncr** [comment]
Looking futher at this it looks like this plugin relies on the `$ZSH` variable to point to `~/.oh-my-zsh` as per described [here](https://github.com/robbyrussell/oh-my-zsh#custom-directory).

I believe you have `ZSH` set up to `~/.oh-my-zsh` that's the reason it doesn't fail in that shell session/tab. You should add the following line to `.zshrc`:

    export ZSH="~/.antigen/repos/https-COLON--SLASH--SLASH-github.com-SLASH-robbyrussell-SLASH-oh-my-zsh.git/"

For this plugin to work correctly.

**Comment by desyncr** [comment]
Whoops. Actually yes, it's an issue with the cache library. Should be fixed with #400.

**Comment by desyncr** [comment]
Should be fixed on `develop`, ready for `v1.4.0`.

Thanks for reporting.
